library/ajax/log_print_action_ajax.php

```diff
@@ -14,16 +14,12 @@
 
 require_once("../../interface/globals.php");
 
+use Html2Text\Html2Text;
 use OpenEMR\Common\Csrf\CsrfUtils;
 use OpenEMR\Common\Logging\EventAuditLogger;
 
 if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
     CsrfUtils::csrfNotVerified();
 }
 
-$instance = new html2text($_POST['comments']);
-$h2t = &$instance;
-$h2t->width = 0;
-$h2t->_convert(false);
-
-EventAuditLogger::instance()->newEvent("print", $_SESSION['authUser'], $_SESSION['authProvider'], 1, $h2t->get_text());
+EventAuditLogger::instance()->newEvent("print", $_SESSION['authUser'], $_SESSION['authProvider'], 1, (new Html2Text($_POST['comments']))->getText());
```

Comment:
![2DV](https://user-images.githubusercontent.com/278968/148142396-45c4dbc0-b2f4-4a05-b57a-2d2f67720eaa.gif)

Reply:
actually, will think will also need the following options in there:
EventAuditLogger::instance()->newEvent("print", $_SESSION['authUser'], $_SESSION['authProvider'], 1, (new Html2Text($_POST['comments']), ['do_links' => 'none', 'width' => 0])->getText());

Reply:
thanks for looking up those [options](https://github.com/mtibben/html2text/blob/master/src/Html2Text.php#L215-L225)